packages/features/bookings/Booker/components/EventMeta.tsx

```diff
@@ -175,7 +180,10 @@ export const EventMeta = ({
                       container: () => "max-w-full",
                     }}
                     value={timezone}
-                    onChange={(tz) => setTimezone(tz.value)}
+                    onChange={({ value }) => {
+                      setTimezone(value);
+                      setBookerStoreTimezone(value);
+                    }}
```

Comment:
In case you change timezone on booker page, update both bookerStore timezone and preferredTime that is being stored in localStorage